src/test/java/com/dabsquared/gitlabjenkins/connection/GitLabConnectionConfigTest.java

```diff
@@ -101,7 +101,7 @@ public void doCheckConnection_forbidden() {
 
     @Test
     public void doCheckConnection_proxy() {
-        jenkins.getInstance().proxy = new ProxyConfiguration("0.0.0.0", 80);
+        jenkins.getInstance().proxy = new ProxyConfiguration("0.0.0.0", 47);
```

Comment:
```suggestion
        jenkins.getInstance().proxy = new ProxyConfiguration("0.0.0.0", 80);
```
Can we revert this change since I am not comfortable to change this if it works for us?